src/components/loop-form.tsx

```diff
@@ -93,46 +126,46 @@ export const RobotLoopForm = (props: robotLoopFormProps) => {
           label="Number of loops"
           variant="outlined"
           error={!!numLoopsError}
+          helperText={numLoopsError}
         />
-        {numLoopsError && (
-          <p id="numLoopsError" className={classes.error}>
-            {numLoopsError}
-          </p>
-        )}
       </div>
 
       <div className={classes.divForm}>
         <Autocomplete
           getOptionLabel={option => option}
           onChange={(e, value) => setStartLocation(value || '')}
-          options={listOfPlaces}
+          options={fakePlaces()[targetFleetName]}
```

Comment:
Maybe it's a good idea for this to be a `state` or a `Memo`, because in the near future we need to replace this for the real value obtained by a service.

E.g:
```
const listOfPlaces = React.useMemo(()=>fakePlaces()[targetFleetName] )
or 
const [listOfPlaces, setListOfPlaces] = useState([])

useEffect(()=>{
    // When we have the service we will need to modify just this function
    setListOfPlaces.set(fakePlaces()[targetFleetName])
}, [targetFleetName])

options={listOfPlaces}
```

Let me know what you think about it

Reply:
I was struggling with this a lot actually! so previously `listOfPlaces` was passed as a string array into the LoopForm because the RobotItem would have created `listOfPlaces` with `const listOfPlaces = fakePlaces()[fleetName]` so I was not sure how to keep the listOfPlaces variable when moving the loop form out of the robots panel.
but what you've suggested seems viable, and I agree that it should be easily changed when the real value from the service is made use of.